pkgs/top-level/all-packages.nix

```diff
@@ -23151,8 +23151,16 @@ with pkgs;
 
   libtifiles2 = callPackage ../development/libraries/libtifiles2 { };
 
-  libtiff = callPackage ../development/libraries/libtiff { };
-  libtiff_4_5 = callPackage ../development/libraries/libtiff/4.5.nix { };
+  inherit
+    ({
+      libtiff = callPackage ../development/libraries/libtiff { };
+      libtiff_4_5 = callPackage ../development/libraries/libtiff/4.5.nix { };
+      libtiff_and_tools = callPackage ../development/libraries/libtiff/libtiff_and_tools.nix { };
+    })
+    libtiff
+    libtiff_4_5
+    libtiff_and_tools
```

Comment:
The name should probably be libtiff_t otherwise this could also be the variant of the normal libraries but with enabled tools.

Reply:
I like `libtiff_t` as well. 

@SuperSandro2000 , thanks for your review. You are doing a great job !

Reply:
Done.  Also rebased onto current `nixos-unstable`.

Reply:
Thanks.